### PR TITLE
feat(admin-ui): group route activities

### PR DESCRIPTION
**Pull request by lundeeell**

### CODE REVIEW

File: packages/engine-ui/src/components/RouteActivities.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            {activityGroup.map((activity, i) => (
+              <Elements.RoundedLink
+                key={i}
+                onMouseOver={() =>
+                  dispatch({ type: 'highlightBooking', payload: activity.id })
+                }
+                onMouseLeave={() =>
+                  dispatch({ type: 'highlightBooking', payload: undefined })
+                }
+                to={`/plans/routes/${route.id}/booking/${activity.id}`}
```
Comment (momentiris) [inline packages/engine-ui/src/components/RouteActivities.tsx:100]: This doesn't seem to be working. I only get to a blank sidebar.

File: packages/engine-ui/src/components/RouteActivities.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+import styled from 'styled-components'
+import { Activity, Route } from '../types'
+import helpers from '../utils/helpers'
+import { UIStateContext } from '../utils/UIStateContext'
+import Elements from './Elements'
+
+const ActivityGroup = styled.div`
+  margin-left: auto;
+  display: flex;
+  flex-direction: column;
```
Comment (momentiris) [inline packages/engine-ui/src/components/RouteActivities.tsx:12]: I think it's a bit more clear with some added space between the upphämtning and avlämning.
```suggestion
  flex-direction: column;
  margin-bottom: 2rem;
```

File: packages/engine-ui/src/components/RouteActivities.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+  height: 100%;
+`
+
+interface Props {
+  route: Route
+}
+
+const groupByLocation = (activities: Activity[]): [Activity[]] => {
+  const { data } = activities.reduce(
+    (prev: any, curr: any) => {
```
Comment (momentiris) [inline packages/engine-ui/src/components/RouteActivities.tsx:47]: Can we type these as Activity?

File: packages/engine-ui/src/components/RouteActivities.tsx (hunk 1 above)
Comment (lundeeell) [reply packages/engine-ui/src/components/RouteActivities.tsx:100]: That is an error that was introduced in https://github.com/Iteam1337/predictivemovement/pull/203

File: packages/engine-ui/src/components/RouteActivities.tsx (hunk 3 above)
Comment (lundeeell) [reply packages/engine-ui/src/components/RouteActivities.tsx:47]: Will look into typings

**Review by momentiris — APPROVED** [review]

**Pull request merged**